rt_gene/launch/estimate_gaze.launch

```diff
@@ -25,7 +25,8 @@
 
       <node pkg="rt_gene" type="extract_landmarks_node.py" name="extract_landmarks_new" output="screen">
           <param name="device_id_facedetection"  value="$(arg device_id_facedetection)" />
-          <param name="device_id_facealignment"  value="$(arg device_id_facealignment)" /> 
+          <param name="device_id_facealignment"  value="$(arg device_id_facealignment)" />
+          <param name="face_encoding_tracker" value="True" />
```

Comment:
Should be `False`?

Reply:
Agreed - must have committed while testing